app/client-v2/src/components/ProjectView/MicroReactColumn.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { useProjectStore } from "@/stores/projectStore";
+import { type ProjectSample, AnalysisType } from "@/types/projectTypes";
+
+const store = useProjectStore();
+const props = defineProps<{
+  data: ProjectSample;
+}>();
+</script>
+
+<template>
+  <div v-if="store.project.status?.microreact === 'finished'" class="flex gap-2">
```

Comment:
I wonder if we shold have an enum for statuses like "finished" rather than using magic strings..

Reply:
i have used union types for this statusus already :smile:

Reply:
Yeah ok, less nice but more efficient!